src/include/cache.i

```diff
@@ -91,6 +91,48 @@ __wt_cache_dirty_inuse(WT_CACHE *cache)
 }
 
 /*
+ * __wt_cache_status --
+ *	Return if the cache usage exceeds the eviction or dirty targets.
+ */
+static inline void
+__wt_cache_status(WT_SESSION_IMPL *session, int *evictp, int *dirtyp)
```

Comment:
How would you feel about moving the ```WT_EVICT_PASS_XXX``` flags into the connection handle and basing decisions based on that flag, rather than sharing a check that needs to do actual work around the code?

I'm hoping to move the eviction code towards being more of a state machine, defined by the WT_EVICT_PASS flags, and shuffling this bit of that decision making code into a separate function makes that feel further away.